src/EditorFeatures/CSharpTest/Diagnostics/AddUsing/AddUsingTests.cs

```diff
@@ -1897,171 +1897,67 @@ static void Main(string[] args)
 @"using lowercase ; namespace A { class A { static void Main ( string [ ] args ) { var a = new b ( ) ; } } } namespace lowercase { class b { } } namespace Uppercase { class B { } } ");
         }
 
-        public partial class AddUsingTestsWithAddImportDiagnosticProvider : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
```

Comment:
Diff looks awful.  All that happened was that some tests moves out of AddUsingTestsWithAddImportDiagnosticProvider and moved into the outer "AddUsingTests" class.  These tests no longer need the specialized add-using logic now that Neal has made the compiler report more errors within lambdas.

Note: not all errors in lambdas are reported.  I have reactivated the bug about that.  I've also opened https://github.com/dotnet/roslyn/issues/7536 so that the compiler will report errors in incomplete members so we see AddUsing work for that as well.